### PR TITLE
Task5 Threading

### PR DESCRIPTION
**Pull request by GOVINDDIXIT**
Link:https://code.hackerearth.com/b89fd6C

### CODE REVIEW

**Comment by GOVINDDIXIT** [comment]
@mubaris pls review it

File: Solutions/GOVINDDIXIT/Task4/sine_curve.py [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import matplotlib.pyplot as plt
```
Comment (mubaris) [inline Solutions/GOVINDDIXIT/Task4/sine_curve.py]: Remove this file. Your task 4 PR is pending.

**Review by mubaris — CHANGES_REQUESTED** [review]

File: Solutions/GOVINDDIXIT/Task4/sine_curve.py (hunk 1 above)
Comment (GOVINDDIXIT) [reply Solutions/GOVINDDIXIT/Task4/sine_curve.py]: I had send the pr from different branches

File: Solutions/GOVINDDIXIT/Task4/sine_curve.py [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+import numpy as np
+
+x=np.arange(0.0,2.0,0.04)
```
Comment (mubaris) [inline Solutions/GOVINDDIXIT/Task4/sine_curve.py]: Update the range from 0 to 10

and `y = 2 sin (pi * x / 4)`

**Review by mubaris — CHANGES_REQUESTED** [review]

**Comment by GOVINDDIXIT** [comment]
is this PR ok?  i am confused

**Comment by GOVINDDIXIT** [comment]
@mubaris I am closing this i will send PR again one by one

**Pull request closed without merging**